test/Eris/Generator/SuchThatGeneratorTest.php

```diff
@@ -20,6 +20,18 @@ function($n) { return $n % 2 == 0; },
         );
     }
 
+    public function testAcceptsPHPUnitConstraints()
+    {
+        $generator = new SuchThatGenerator(
+            $this->callback(function($n) { return $n % 2 == 0; }),
+            ConstantGenerator::box(10)
+        );
+        $this->assertEquals(
```

Comment:
@giorgiosironi 

How about using a more strict assertion here?

After all, this would also pass if `$generator->__invoke($this->size)->unbox()` evaluated to `'3'`.